.github/scripts/release.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+
+# build artifacts
+make distribution
```

Comment:
Is this the make file in the project root?

Reply:
Ah yes. As you can probably tell I haven't actually tested this all-in-one script yet - it was just to show the concept.